src/timeout.ts

```diff
@@ -150,6 +150,11 @@ function isCSOTTimeoutContextOptions(v: unknown): v is CSOTTimeoutContextOptions
 
 /** @internal */
 export abstract class TimeoutContext {
+  start: number;
```

Comment:
This likely belongs in the CSOTTimeoutContext specifically, since it's not used in the abstract base class or in the legacy timeout context.

Reply:
Good call, moved!